### PR TITLE
add a standalone subscribe page

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu, @ellenkorbes,

Please review the following commits I made in branch nicks/subscribe:

57e54ead1ec04df5e9b6e97ddb346182dc32d0a7 (2021-06-29 15:20:43 -0400)
add a standalone subscribe page

20730ce47a28e86ba0bbedb0f323fc0fa5cb1f51 (2021-06-29 14:50:02 -0400)
site: convert all subscribe forms to mailchimp

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Screenshot from 2021-06-29 15-23-21

File: src/subscribe.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+layout: enterprise
+title: Tilt Dev News
+---
+
+<h2 class="Enterprise-heroTitle">Tilt Dev News</h2>
+<section class="Enterprise-hero">
+  <div class="Enterprise-hero-text">
+    <p class="Enterprise-hero-text-subhead">
+      When you subscribe to our roughly-once-a-month mailing list, you get:
```
Comment (ellenkorbes) [inline src/subscribe.md:10]:
```suggestion
      When you subscribe to our roughly-once-or-twice-a-month mailing list, you get:
```

Comment (ellenkorbes) [reply src/subscribe.md:10]: We're publishing more, smaller things lately. I don't want us sending huge newsletters, so we'll probably be publishing closer to twice a month.

File: src/subscribe.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    </p>
+    <div class="Enterprise-hero-text-detail">
+      <ul>
+        <li>Multi-service dev emotional support.</li>
```
Comment (ellenkorbes) [inline src/subscribe.md:14]:
```suggestion
        <li>Multi-service development emotional support.</li>
```

"Dev" made me read it as "developer" at first glance 🤔

**Review by ellenkorbes — APPROVED** [review]
Love it!

File: src/subscribe.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+layout: enterprise
+title: Tilt Dev News
+---
+
+<h2 class="Enterprise-heroTitle">Tilt Dev News</h2>
```
Comment (ellenkorbes) [inline src/subscribe.md:6]:
```suggestion
<h2 class="Enterprise-heroTitle">Tilt News</h2>
```

Is our brand 'Tilt' or 'Tilt Dev'? 🤔

Comment (nicks) [reply src/subscribe.md]: lately i tend to use "Tilt Dev", simply because it's easier to google

Comment (ellenkorbes) [reply src/subscribe.md]: Sounds good

**Pull request merged**